src/controllers/likeController.js

```diff
@@ -0,0 +1,24 @@
+const asyncHandler = require("express-async-handler");
+const Location = require("../models/locationModel");
+
+const addLike = asyncHandler(async (req, res) => {
+  const location = await Location.findById(req.params.id);
+
+  if (!location) {
+    res.status(400);
+    throw new Error("Location not found");
```

Comment:
Maybe it makes more sense to put a return here, since the throw won't propagate to the client anyway?